src/main/java/seedu/duke/Duke.java

```diff
@@ -17,18 +21,30 @@
 public class Duke {
     private static Logger logger = Logger.getLogger("Run Duke Log");
     private Ui ui;
+
     private FoodList foodList;
 
+    private StorageFile storageFile;
+
     public Duke() {
         ui = new Ui();
-        foodList = new FoodList();
+        try {
+            storageFile = new StorageFile();
+        } catch (InvalidStorageFilePathException e) {
+            System.out.println("File path is invalid");
+        }
+
+        try {
+            foodList = storageFile.load();
+        } catch (StorageOperationException e) {
+            System.out.println("Unable to read save file");
```

Comment:
I think the ui method `showError` can be used instead of `System.out.println`

Reply:
Alright, will change that.